DropDown/src/DropDown.swift

```diff
@@ -635,16 +657,13 @@ extension DropDown {
 	}
 	
 	private func fittingWidth() -> CGFloat {
-		struct Static {
-			static let templateCell = DropDownCell.Nib.instantiateWithOwner(nil, options: nil)[0] as! DropDownCell
-		}
-		
+		let templateCell = cellNib.instantiateWithOwner(nil, options: nil)[0] as! DropDownCell
```

Comment:
`fittingWidth` was called from 1 to 3 times for each `show` of the `DropDown` and each orientation change and keyboard (dis)appearance 😉